backstop/pages.json

```diff
@@ -1,6 +1,32 @@
 [
   {"label": "Homepage", "url": "/"},
   {"label": "HomepageWithAlert", "url":  "/", "showAlerts":  true, "readySelector": "span.ma__emergency-alert__time-stamp", "delay": 2000},
+  {"label": "Homepage Login link (Large sizes)", "delay": 2000, "url": "/", "clickSelector": ".ma__header__hamburger__utility-nav .ma__utility-nav__items li.ma__utility-nav__item:last-child button.ma__utility-nav__link", "viewports": [
+    {
+      "label": "desktop",
+      "width": 1920,
+      "height": 1080
+    },
+    {
+      "label": "tablet",
+      "width": 1024,
+      "height": 768
+    }
+  ] },
+  {"label": "Homepage Login link (Mobile)", "delay": 2000, "url": "/", "viewports": [
+    {
+      "label": "phone",
+      "width": 320,
+      "height": 480
+    }
+  ] },
+  {"label": "Homepage Login link (Mobile)", "delay": 2000, "url": "/", "viewports": [
```

Comment:
@arthurbaghdas Looks like we are duplicating the mobile viewport

Reply:
Good catch, resolved duplicates. Thanks